newsfragments/430.minor

```diff
@@ -0,0 +1 @@
+Add your info here
```

Comment:
We should be following the dev-guide and filing tickets, _then_ doing PRs for those tickets. If you want to propose changes, please propose a change to the dev guide.

https://github.com/LeastAuthority/magic-folder/blob/main/docs/development.rst

Reply:
This is part of #210, but split out into a separate PR to be easier to review.

Reply:
that's fine, but having PRs point newsfragments at themselves isn't really the point. i get that it's annoying sometimes, but that's what we've got ...